conf/application.conf

```diff
@@ -84,7 +84,7 @@ controllers {
     needsLogging = false
     needsAuditing = false
   }
-  # TODO this needs to be updated to the new auth system (maybe)
+
```

Comment:
Can you remove the config lines from 103-119 as they aren't required

Reply:
Done.